### PR TITLE
hpcc4j-358 Fixed deprecated Constructors

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner <Michael.Gardner@lexisnexisrisk.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).

## Checklist:
- [x] I have created a corresponding JIRA ticket for this submission
- [x] My code follows the code style of this project.
  - [ ] I have applied the Eclipse code-format template provided.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the HPCC Systems CONTRIBUTORS document (https://github.com/hpcc-systems/HPCC-Platform/wiki/Guide-for-contributors).
- [ ] The change has been fully tested:
  - [ ] I have performed unit tests to cover my changes.
  - [ ] I have performed system test and covered possible regressions and side effects.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.

### CODE REVIEW

File: wsclient/src/main/java/org/hpccsystems/ws/client/platform/Workunit.java [hunk 1]
```diff
@@ -690,7 +690,7 @@ void fullRefresh(boolean includeGraphs, boolean includeResults, boolean includeS
             HPCCWsWorkUnitsClient wsWorkunitsClient = platform.getWsClient().getWsWorkunitsClient();
             try
             {
-                WorkunitWrapper wi = wsWorkunitsClient.getWUInfo(info.getWuid(), includeResults, includeGraphs, includeSourceFiles, includeApplicationValues, false, false, false, false, false);
+                WorkunitWrapper wi = wsWorkunitsClient.getWUInfo(info.getWuid());
```
Comment (rpastrana) [inline wsclient/src/main/java/org/hpccsystems/ws/client/platform/Workunit.java:693]: This PR might stil be WIP, but this change could change behavior, the includeResults, includeGraphs, includeSourceFiles, includeApplicationValues params would be ignored.

Comment (rpastrana) [reply wsclient/src/main/java/org/hpccsystems/ws/client/platform/Workunit.java:693]: Correct, we deprecated in candidate-7.10.0, the alternative is getWUInfo(WSInfoRequestWrapper)
So we need to call it like this:
WUInfoRequestWrapper wuinforeq = new WUInfoRequestWrapper();
            	wuinforeq.setWuid(info.getWuid());
            	wuinforeq.setIncludeResults(includeResults);
            	wuinforeq.setIncludeGraphs(includeGraphs);
            	wuinforeq.setIncludeSourceFiles(includeSourceFiles);
            	wuinforeq.setIncludeApplicationValues(includeApplicationValues);
            	wuinforeq.setIncludeDebugValues(false);
            	wuinforeq.setIncludeExceptions(false);
            	wuinforeq.setIncludeVariables(false);
            	wuinforeq.setIncludeXmlSchemas(false);
            	wuinforeq.setIncludeTimers(false);
            	
                WorkunitWrapper wi = wsWorkunitsClient.getWUInfo(wuinforeq);

Comment (rpastrana) [reply wsclient/src/main/java/org/hpccsystems/ws/client/platform/Workunit.java:693]: @Michael-Gardner bump above comment

**Comment by Michael-Gardner** [comment]
@rpastrana that final change was made for the WUInfoRequest wrapper

File: wsclient/src/main/java/org/hpccsystems/ws/client/platform/Workunit.java [hunk 2]
```diff
@@ -690,7 +691,18 @@ void fullRefresh(boolean includeGraphs, boolean includeResults, boolean includeS
             HPCCWsWorkUnitsClient wsWorkunitsClient = platform.getWsClient().getWsWorkunitsClient();
             try
             {
-                WorkunitWrapper wi = wsWorkunitsClient.getWUInfo(info.getWuid(), includeResults, includeGraphs, includeSourceFiles, includeApplicationValues, false, false, false, false, false);
+            	WUInfoRequestWrapper wuinforeq = new WUInfoRequestWrapper();
```
Comment (rpastrana) [inline wsclient/src/main/java/org/hpccsystems/ws/client/platform/Workunit.java:694]: looks reasonable, but the indent seems a little off @Michael-Gardner

File: wsclient/src/main/java/org/hpccsystems/ws/client/platform/Workunit.java [hunk 3]
```diff
@@ -690,7 +691,18 @@ void fullRefresh(boolean includeGraphs, boolean includeResults, boolean includeS
             HPCCWsWorkUnitsClient wsWorkunitsClient = platform.getWsClient().getWsWorkunitsClient();
             try
             {
-                WorkunitWrapper wi = wsWorkunitsClient.getWUInfo(info.getWuid(), includeResults, includeGraphs, includeSourceFiles, includeApplicationValues, false, false, false, false, false);
+                WUInfoRequestWrapper wuinforeq = new WUInfoRequestWrapper();
```
Comment (Michael-Gardner) [inline wsclient/src/main/java/org/hpccsystems/ws/client/platform/Workunit.java:694]: fixed the indentation issue here

Comment (Michael-Gardner) [reply wsclient/src/main/java/org/hpccsystems/ws/client/platform/Workunit.java:694]: @rpastrana should be good to merge

**Review by rpastrana — APPROVED** [review]
@Michael-Gardner approved

**Pull request merged**